static/js/pivotal.js

```diff
@@ -197,7 +253,7 @@
             for (pt_id in pivotal_ids) {
               var id = pivotal_ids[pt_id];
 
-              getPivotalInfo(PIVOTAL_TOKEN, id, function(info) {
+              getPivotalStoryInfo(PIVOTAL_TOKEN, id, function(info) {
                 display_message[id] = info;
                 injectPivotalStatus(project, display_message);
```

Comment:
Sound good to me :+1: Don't think it's an issue at the moment but can optimize it :smile: